### PR TITLE
Simplify the wording of the manifesto

### PR DESCRIPTION
**Pull request by simbs**
Edits following the discussion from #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,87 +1,26 @@
-# The Good Data Movement manifesto.
+# The Good Data Movement is a campaign group for fair, ethical and non-exploitative use of data.
+We are working towards a future where people’s personal data is used for the benefit of everyone, consent to it’s collection and use is a well informed decision that will be respected, and people receive a share of the value their data produces.
```
Comment (ps-george) [inline README.md]: its not it's

**Review by ps-george — APPROVED** [review]

**Comment by ps-george** [comment]
Sorry I approved it but actually it's _does_ need to be changed to its before merging 😅

File: README.md (hunk 1 above)
Comment (simbs) [inline README.md]:
```suggestion
We are working towards a future where people’s personal data is used for the benefit of everyone, consent to its collection and use is a well informed decision that will be respected, and people receive a share of the value their data produces.
```
